test/blocks/marquee-container/marquee-container.test.js

```diff
@@ -0,0 +1,17 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+describe('Marquee container', () => {
+  before(async () => {
+    const module = await import('../../../libs/blocks/marquee-container/marquee-container.js');
+    const containers = document.body.querySelectorAll('.marquee-container');
+    containers.forEach((container) => {
+      module.default(container);
+    });
```

Comment:
This can be simpler 😊
```suggestion
    document.body.querySelectorAll('.marquee-container').forEach((container) => module.default(container));
```